kalite/knowledgemap/api_resources.py

```diff
@@ -0,0 +1,38 @@
+from tastypie import fields
+from tastypie.exceptions import Unauthorized
+from tastypie.resources import ModelResource
+
+from kalite.shared.decorators import get_user_from_request
+
+from kalite.facility.api_resources import FacilityUserResource
+from kalite.shared.api_auth import UserObjectsOnlyAuthorization
+from kalite.main.models import ExerciseLog
+
+class ExerciseSummaryResource(ModelResource):
```

Comment:
I am confused. Why are there changes to the coachreport endpoint above when you have another endpoint here?